From 650633d4e956dc6b628da30c868e39fba76a0b29 Mon Sep 17 00:00:00 2001
From: Steve Dickson <steved@redhat.com>
Date: Mon, 9 Feb 2015 08:54:54 -0500
Subject: [PATCH] rpc-statd-notify: Don't wait for the network to come up

rpc-statd-notify only needs to wait for the network
interface to be configured not to be connected since
it forks into background and will wait 15mins for
the network to come up.

This decrease boot times when networks are not
and will not be connected.

Signed-off-by: Steve Dickson <steved@redhat.com>
Reported-By: Eric Work <work.eric@gmail.com>
---
 systemd/rpc-statd-notify.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/systemd/rpc-statd-notify.service b/systemd/rpc-statd-notify.service
index 941afe5..a655445 100644
--- a/systemd/rpc-statd-notify.service
+++ b/systemd/rpc-statd-notify.service
@@ -1,7 +1,7 @@
 [Unit]
 Description=Notify NFS peers of a restart
 Requires=network-online.target
-After=network-online.target nss-lookup.target
+After=network.target nss-lookup.target
 
 # if we run an nfs server, it needs to be running before we
 # tell clients that it has restarted.
-- 
2.1.0

